Language/Functions/Digital IO/digitalRead.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 title: digitalRead()
-categories: [ "Functions" ]
-subCategories: [ "Digital I/O" ]
+categories: [ "Funções" ]
```

Comment:
we need only subcategories to be translated, keep 'functions' here, thanks

Reply:
Alright, got it.